README.md

```diff
@@ -202,6 +202,37 @@ MainScope mainScope = new MainScopeImpl();
 ChildScope childScope = mainScope.child();
 ```
 
+## Dependency Resolution and Exposure
+
+Motif requires explicit dependency declarations to properly resolve and inject dependencies across scopes. When a parent scope needs to provide a dependency to a child scope, the dependency must be explicitly exposed using the `@Expose` annotation.
```

Comment:
Doesn't line ~140 of README roughly mentions it? Or was that presentation done in a confusing way?

I agree we perhaps need to make the use of `@Expose` a more pronounced section given how important it is, but perhaps we can remove / restructure the existing paragraph?

Reply:
Totally fine with restructuring the existing paragraph

Reply:
Updated, thoughts? I think the explicit dependency declaration requirement is what is not completely clear